spacy/tests/test_cli_app.py

```diff
@@ -40,3 +44,130 @@ def test_benchmark_accuracy_alias():
     assert result_benchmark.stdout == result_evaluate.stdout.replace(
         "spacy evaluate", "spacy benchmark accuracy"
     )
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "component,examples",
+    [
+        (
+            "tagger",
+            [
+                dict(words=["I", "like", "cats"], tags=["PRP", "VBP", "NNS"]),
+                dict(words=["I", "like", "dogs"], tags=["PRP", "VBP", "NNS"]),
+            ],
+        ),
```

Comment:
I see the point for having this test parameterized, but maybe we can store the example data in global vars at the top? It'll make the test itself more readable and the data samples reusable :-)

Reply:
Yes, still parameterized.

Reply:
Alright, I've done a first pass at this. I placed the example data next to these tests specifically, as I imagine as this file grows with other tests it might be helpful that they're located next to the test they're used. I can shift them to the top of the file as well, if you think that's a better place for them for now too.